modules/sdk/gradle-plugins/src/com/liferay/gradle/plugins/LiferayWebAppPlugin.java

```diff
@@ -67,6 +68,29 @@ public void apply(Project project) {
 		configureWebAppDirName(project);
 	}
 
+	protected DirectDeployTask addTaskDirectDeploy(Project project) {
+		DirectDeployTask directDeployTask = GradleUtil.addTask(
+			project, DIRECT_DEPLOY_TASK_NAME, DirectDeployTask.class);
+
+		directDeployTask.dependsOn(WarPlugin.WAR_TASK_NAME);
+
+		directDeployTask.setDescription(
+			"Processes the war archive just like it was run through Liferay " +
```

Comment:
I copied the description from here: https://github.com/mikakoivisto/liferay-maven-incubation/wiki/Plugin-Direct-Deployer